ingest/ingest_pipeline.py

```diff
@@ -166,17 +179,23 @@ def load_cell_metadata(self):
                 subcollection_doc = self.cell_metadata.data_subcollection[annotation]
                 doc_ref_sub = doc_ref.collection(subcollection_name).document()
                 doc_ref_sub.set(subcollection_doc)
-            except exceptions.InvalidArgument as e:
+            except exceptions.InvalidArgument:
                 # Catches invalid argument exception, which error "Maximum
                 # document size" falls under
-                print(e)
                 batch = self.db.batch()
                 for subdoc in self.cell_metadata.chunk_subdocuments(
                     doc_ref_sub.id, doc_ref_sub._document_path, annotation
                 ):
                     batch.set(doc_ref_sub, subdoc)
 
                 batch.commit()
+            # An unexpected exception has been encountered
+            except Exception as e:
+                print(e)
+                # At this point another exception has occured
+                # TODO: Implement deletion of loaded documents
```

Comment:
It may be more expedient instead to halt execution, have the Ingest run return the error, and then let Rails do the cleanup.

Reply:
Yeah once we've addressed how to return errors back to rails I think that's fair.